src/app/app.module.ts

```diff
@@ -202,4 +209,10 @@ import { MomentDatePipe } from './pipes/moment-date.pipe';
 	bootstrap: [AppComponent],
 	entryComponents: moduleComponents
 })
-export class AppModule { }
+export class AppModule {
+	constructor(library: FaIconLibrary) {
+		// Add an icon to the library for convenient access in other components
+		library.addIconPacks(fas, far, fab);
+		library.addIcons(faTwitter);
```

Comment:
Since you used the Twitter one manually for the module thing, do you need to explicitly add it here?